lib/fixup.rb

```diff
@@ -189,3 +189,25 @@ def self.remove; end
 
 # Remove pagerenv tmp file in CREW_PACKAGES_PATH if it exists
 FileUtils.rm "#{CREW_PACKAGES_PATH}/pagerenv" if File.file?("#{CREW_PACKAGES_PATH}/pagerenv")
+
+# Handle broken system glibc affecting gcc_lib on newer x86_64 ChromeOS milestones.
+if (ARCH == 'x86_64') && (LIBC_VERSION.to_f >= 2.36)
+  abort("patchelf is needed. Please run: 'crew install patchelf && crew update'") unless File.file?(File.join(CREW_PREFIX, 'bin/patchelf'))
+  # Link the system libc.so.6 to also require our renamed libC.so.6
+  # which provides the float128 functions strtof128, strfromf128,
+  # and __strtof128_nan.
+  @libc_patch_libraries = %w[libstdc++.so.6]
```

Comment:
This also does not need to be an instance variable, and it probably doesn't need to be an array.

Reply:
The array is there in case we have to add more libraries.